testassets/InteropTestsClient/InteropTestsClient.csproj

```diff
@@ -44,8 +40,15 @@
     <PackageReference Include="System.Net.Http.WinHttpHandler" Version="$(SystemNetHttpWinHttpHandlerPackageVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0' OR '$(TargetFramework)' == 'net6.0' OR '$(TargetFramework)' == 'net7.0'">
+  <ItemGroup Condition="'$(PublishAot)'!='true'">
+    <ProjectReference Include="..\..\src\Grpc.Auth\Grpc.Auth.csproj" />
+    <ProjectReference Include="..\..\src\Grpc.Net.Client\Grpc.Net.Client.csproj" />
     <ProjectReference Include="..\..\src\Grpc.Net.Client.Web\Grpc.Net.Client.Web.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(PublishAot)'=='true'">
+    <PackageReference Include="Grpc.Auth" Version="$(GrpcDotNetPackageVersion)" />
```

Comment:
What's the reason for this?

Reply:
This bug - https://github.com/dotnet/sdk/issues/28169